aeronet/dataset/transforms/_vectorize.py

```diff
@@ -69,7 +69,7 @@ def _vectorize(binary_image, epsilon=0., min_area=1., transform=IDENTITY, upscal
         binary_image = cv2.resize(binary_image, (int(w * upscale), int(h * upscale)), cv2.INTER_NEAREST)
 
     # search for all contours
-    image, contours, hierarchy = cv2.findContours(
+    contours, hierarchy = cv2.findContours(
```

Comment:
Let`s make in with backward compatability:
```python
output = cv2.findContours(...)
contours, hierarchy = output if len(output) == 2 else output[1:]  # OpenCV 3 compatability
```

Reply:
Updated, tested to work with both function signatures.